back/app/controllers/web_api/v1/reset_password_controller.rb

```diff
@@ -16,6 +16,8 @@ def reset_password_email
     reset_password_service.send_email_later user, token
     reset_password_service.log_activity user, token
     head :accepted
+  rescue ActiveRecord::RecordNotFound
```

Comment:
I think I would rather compare `user` to `nil` without rescuing an error to make it explicitly clear what exactly can be `NotFound`.
```ruby
  user = User.not_invited.find_by_cimail(params[:user][:email])
  if user
    reset_password_service = ResetPasswordService.new
    token = reset_password_service.generate_reset_password_token user
    user.update! reset_password_token: token
    reset_password_service.send_email_later user, token
    reset_password_service.log_activity user, token
  end
  head :accepted
```
But the current version is fine too.